frontend/src/types/ResidentTypes.ts

```diff
@@ -1,13 +1,17 @@
 export type Resident = {
-  id?: number;
-  residentId?: string;
+  id: number;
+  residentId: string;
   initial: string;
   roomNum: number;
   dateJoined: Date;
   dateLeft?: Date;
   building: string;
-  label?: string;
-  value?: number;
+};
+
+export type ResidentLabel = {
+  id: number;
```

Comment:
Noticed this in a couple places, is there a reason why a label has an ID if the ID is typically going to be the value?

Reply:
seems like the same is done for userLabel:
```
export type UserLabel = {
  id: number;
  label: string;
  value: number;
};
```

Reply:
not really, took id out